Subspaces improvements (#521)
desmos-labs/desmos: docs/developers/types/staging/subspaces/subspace.md

SHA-256 hash string.

## `Name`

> The `Name` is the human-readable name of the subspace. It must be a non-empty or blank string.

Reviewer: ```suggestion
The `Name` is the human-readable name of the subspace. It must be non-empty nor blank.
```

## `Description`

> The `Description` contains a brief summary of what's the subspace about.

Reviewer: ```suggestion
The `Description` contains a brief summary of what the subspace is about.
```
